src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,19 @@
 const accessibilityQuiz = [
+  {
+    Question:
+      "When creating charts, what should be added so that color is not the only identifier to convey the meaning of the information?",
+    Answer:
+      "Patterns.",
+    Distractor1:
+      "Hint Buttons.",
+    Distractor2:
+      "Gradients.",
+    Distractor3:
+      "There is no need to add an additional identifier, color is enough.",
+    Explanation:
+      "Colorblind and low vision users have difficulties to perceive the color differences. In addition, when the graph appears in greyscale (e.g. user settings, print), it is impossible to every user to see differences.",
+    Link: "https://dequeuniversity.com/tips/color-to-convey-meaning"
```

Comment:
we only accept links from freeCodeCamp resources, w3.org or wikipedia.
please change the link to one of those resources